### DISCUSSION TITLE
Is there a way to create an issue from Github projects using Issue Templates/Forms?

### DISCUSSION THREAD

**Discussion by DS-Matt** [community/community#47819] [Projects and Issues] [answered]
### Select Topic Area

Question

### Body

Hello All -

We use Github Issues and are implementing Github issue forms (using YAML files) to ensure when we create GitHub issues (which we use for tracking Epics, User Stories, tasks, bugs, refactors, DevOps, etc.) the issue accurately reflects the type of issue we are creating and that all the data/information for the specific type of issue is recorded. When going to a repo and creating a new issue we choose the type of issue template/form we want to use, but when making a "draft" issue and then assigning it to a Repo - it creates an issue but doesn't use an issue template or form.

Do I have this configured incorrectly or is this functionality not currently available? 

If not available, can you add this to a roadmap? 

Thanks!

**Comment by jfbaraky** [comment]
Having the same problem here, but looks like it's not available yet 😞

**Comment by azenMatt** [comment, marked as answer]
This is now available!  https://github.blog/changelog/2023-07-13-github-issues-projects-july-13th-update/
